README_extending_docker.md

```diff
@@ -175,3 +175,99 @@ of that file in `unskript-ctl` Folder.
+schedled to run. Edit the cadence at which the scheduler should run and also change the type of connectors you want the checks to run against.
+
+4. Open your Custom Build Dockerfile From the step [above](#building-custom-docker) And Add the following line before the `CMD` line like so:
+```
+FROM unskript/awesome-runbooks:latest as base
+COPY custom/actions/. /unskript/data/actions/
+COPY custom/runbooks/. /unskript/data/runbooks/
+
+# Copy the populate_credentials.sh file to ./
+COPY populate_credentials.sh .
+RUN chmod +x populate_credentials.sh
+
+# Copy Scheduler file to ./
+COPY scheduler .
+RUN chmod 600 scheduler
+
+CMD ["./start.sh"]
+```
+In the above snippet, we have modfied the `Dockerfile.template` to copy the `scheduler` to docker so that it can register
+the scheduler to run at the desired frequency. 
+
+5. Build the docker as explained above and when your custom docker is booted, it will have the scheduler ready to run!
+
+
+
+### How to get checks run report as Email or as Slack Notification
+
+You can configure unskript-ctl to send a notification when you run the checks. The option `--report` at the end of the `-rc` command
+can be used to send the notification. Here is the receipie on how to get run reports when you run the checks. 
+
+1. Run either your custom docker or pull the latest one from `dockerhub`
+```
+docker run -it $CUSTOM_DOCKER_NAME:$CUSTOM_DOCKER_IMAGE -p 8888:8888
+
+or
+
+docker run -it unskript/awesome-runbooks:latest -p 8888:8888 
+```
+
+2. Connect to the docker instance
+```
+CONTAINER_ID=`docker ps | grep awesome-runbooks | awk '{print $1}'`
+docker exec -it $CONTAINER_ID bash
+```
+
+3. You can use the command `add_notification.sh` to configure `unskript-ctl` to send report once the Checks are run. 
+
+For example, if you want to get Slack Notification, here is the command to follow -
+```
+add_notification.sh -c Slack -u https://hooks.slack.com/services/T12345/B12345/XXXXXXX --channel test-alerting
+```
+> Here `-c` Option is used for creating a new Notification. Options are either Slack or SMTP (email)
+> `-u` Option is the webhook URL of Slack 
+> `--channel` Channel to where the notification should be sent
+
+This snippet shows how to configure SMTP (email) Notification
+
+```
+./add_notification.sh -c SMTP -s smtp.server.com -u username@domain.com -p <password_of_username> -t receiver@example.com
+```
+> Here `-s` Option is to specifiy the SMTP server 
+> `-u` Option is to specify the SMTP username. Note, you need to specificy the username with the domain like username@domain.com 
+> `-p` Option is to specify the Password for the above user. Please note, if you are using Gmail as SMTP server, then make sure you create a App Password and use that for the `-p` option
+> `-t` Optino is to specify the Reciever's Email address to whom the Email should be sent to
+
+
+4. You are not set to receive the report whenever you run the check with the `--report` option
```

Comment:
you are not set???? Could you please go over your README.md thoroughly to rule out basic stuff, spelling mistake, capitalized words n all